diff -bru5 a/src/tintin.h b/src/tintin.h
--- a/src/tintin.h	2020-05-10 12:29:49.670306837 +0200
+++ b/src/tintin.h	2020-05-10 12:28:03.470310200 +0200
@@ -203,11 +203,11 @@
 #define COLOR_COMMAND       "\e[38;5;044m" // "<aee>" // cyan
 #define COLOR_RESET         "\e[0m"        // "<088>" // reset
 #define COLOR_SEPARATOR     "\e[38;5;160m" // "<eaa>" // red
 #define COLOR_STATEMENT     "\e[38;5;040m" // "<aea>" // green
 #define COLOR_STRING        "\e[38;5;188m" // "<eee>" // white
-#define COLOR_TEXT          "\e[0m"        // "<ddd>" // grey
+#define COLOR_TEXT          "\e[0;37;49m"  // Patched for Clessidra
 #define COLOR_TINTIN        "\e[38;5;184m" // "<eea>" // yellow
 
 #define TIMER_UPDATE_INPUT               0
 #define TIMER_UPDATE_SESSIONS            1
 #define TIMER_UPDATE_DELAYS              2
