diff --git a/src/misc.c b/src/misc.c
index cb3ad97..19f3b7f 100644
--- a/src/misc.c
+++ b/src/misc.c
@@ -180,11 +180,11 @@ DO_COMMAND(do_echo)
 
 	if (strip_vt102_strlen(ses, ses->more_output) != 0)
 	{
-		output = str_dup_printf("\n\e[0m%s\e[0m", temp);
+		output = str_dup_printf("\n\e[0m%s", temp);
 	}
 	else
 	{
-		output = str_dup_printf("\e[0m%s\e[0m", temp);
+		output = str_dup_printf("\e[0m%s", temp);
 	}
 
 	add_line_buffer(ses, output, lnf);
diff --git a/src/show.c b/src/show.c
index 642f43e..cba6289 100644
--- a/src/show.c
+++ b/src/show.c
@@ -67,11 +67,11 @@ DO_COMMAND(do_showme)
 
 	if (strip_vt102_strlen(ses, ses->more_output) != 0)
 	{
-		output = str_dup_printf("\n%s%s%s", COLOR_TEXT, arg1, COLOR_TEXT);
+		output = str_dup_printf("\n%s%s", COLOR_TEXT, arg1);
 	}
 	else
 	{
-		output = str_dup_printf("%s%s%s", COLOR_TEXT, arg1, COLOR_TEXT);
+		output = str_dup_printf("%s%s", COLOR_TEXT, arg1);
 	}
 
 	add_line_buffer(ses, output, lnf);
